install-and-configure/install/multi-cluster/federated-etl/aggregator.md

```diff
@@ -117,10 +137,10 @@ kubecostAggregator:
     storageRequest: 512Gi
   resources:
     requests:
-      cpu: 1000m
-      memory: 1Gi
+      cpu: 4
+      memory: 12Gi
     limits:
-      # cpu: 2000m
+      cpu: 6
```

Comment:
I wonder if re-structiring this document to take the same small/medium/large/extra large set of recommendations as in the aggregator tuning google doc would be a better approach? for example, a user with 3 nodes and 150 allocations a day will way overprovision this and kubecost will seem expensive. 

The $1,000,000 question is what is a "larger aggregator deployment"? I wonder if we establish a rough mapping, and ask users to check their container count via the new diagnostic endpoint in the UI. These settings would be good for maybe 20-60k containers per day. Below 10k allocations per day, the prior settings would probably work

Reply:
I'm in agreement with all the above! I'm thinking we just start with details for small/medium environments as that should cover a majority of cases for now.

I'll update this doc to establish a rough mapping for what configs to use based on the number of containers per day. I'll err on the side of overprovisioning rather than underprovisioning.